client/components/tinymce/plugins/insert-menu/menu-items.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import Gridicon from 'components/gridicon';
+import SocialLogo from 'components/social-logo';
+
+const GridiconButton = ( { icon, label } ) => (
+	<div>
+		<Gridicon className="wpcom-insert-menu__menu-icon" icon={ icon } />
+		<span className="wpcom-insert-menu__menu-label">{ label }</span>
+	</div>
+);
+
+const SocialLogoButton = ( { icon, label } ) => (
+	<div>
+		<SocialLogo className="wpcom-insert-menu__menu-icon" icon={ icon } />
+		<span className="wpcom-insert-menu__menu-label">{ label }</span>
+	</div>
+);
+
+export default [
+	{
+		name: 'insert_media_item',
+		icon: 'add-image',
+		item: <GridiconButton icon="image-multiple" label={ 'Add Media' } />,
```

Comment:
Thanks @aduth - translations added in d5419ce

**Note** that we need a string review of `Insert content` because it's currently untranslated. We should review the wording before merging and having people translate that particular message.